splice_machine/src/main/java/com/splicemachine/derby/ddl/AsynchronousDDLWatcher.java

```diff
@@ -76,6 +76,7 @@ public void start() throws IOException {
         if(!checker.initialize(this))
             return; //we aren't a server, so do nothing further
 
+        refresher.getDemarcationPoint();
```

Comment:
I think we can remove this

Reply:
This is needed to initialize a local copy of demarcation point in DDLWatchRefresher